doc/_static/mpl.css

```diff
@@ -220,3 +220,15 @@ div.wide-table table th.stub {
 .sidebar-cheatsheets > img {
     width: 100%;
 }
+
+.sidebar-cheatsheets {
```

Comment:
I also do not understand this change

Reply:
Seems like a redundant merge; these rules are all right above here.